turbo.json

```diff
@@ -188,8 +188,10 @@
     "$CALCOM_LICENSE_KEY",
     "$CALCOM_TELEMETRY_DISABLED",
     "$CALENDSO_ENCRYPTION_KEY",
+    "$SEND_FEEDBACK_EMAIL",
```

Comment:
This was also missing and saw errors somewhere complaining about missing this here, so added it.